deployments/core.go

```diff
@@ -0,0 +1,255 @@
+package deployments
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/fuseml/fuseml/cli/helpers"
+	"github.com/fuseml/fuseml/cli/kubernetes"
+	"github.com/fuseml/fuseml/cli/paas/ui"
+	"github.com/kyokomi/emoji"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Core struct {
+	Debug   bool
+	Timeout int
+}
+
+const (
+	coreDeploymentID        = "fuseml-core"
+	coreDeploymentNamespace = "fuseml-core"
```

Comment:
Unless you plan on using a different namespace for every fuseml component (assuming OFC that "core" isn't the only one), why not use `fuseml` as your namespace ?
For example, docker-registry should be moved in the same namespace at some point, given that it's a built-in "core" component not regulated by a 3rd party helm chart or operator.

Reply:
Well, this just follows current architecture. All other components actually have their own namespace.

But if you want all of them (or at least registry+core) move to shared one, I suggest to do it in followup PR.

Reply:
Right, this feels like something to be done in another PR.